CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+**Enhancements:**
+
+- [#108](github.com/castle/castle-ruby/pull/108) move context classes to theirs own scope, move command validations to their own classes, code cleanup
```

Comment:
`move context and command validation to their own scope and classes, code cleanup`